src/test/resources/application.yml

```diff
@@ -0,0 +1,77 @@
+camel:
```

Comment:
Could not pass tests because of absence properties `camel.component.rabbitmq.*`

Reply:
fixed for username and password